src/interfaces.ts

```diff
@@ -45,4 +45,14 @@ export interface MnemonicOptions {
    * BIP44 index number
    */
   index?: number
+
+  /**
+   * Default path for HD wallet. 1023 by default
+   */
+  defaultPath?: string
```

Comment:
You can add a default value here and change the name to `derivationPath`:
```ts
...
derivationPath = `m/44'/60'/0'/0/`
...
```
The last number is the index that you get from index option.

Reply:
Got it!